_data/package.yml

```diff
@@ -79,28 +79,20 @@ keywords: |
     transformation semigroups, partial permutations, inverse semigroups, Green's relations, free inverse semigroup, partition monoid, bipartitions, Rees matrix semigroups.
 citeas: |
     <p class='BibEntry'>
-    [<span class='BibKey'>Mit18</span>]   <b class='BibAuthor'>Mitchell, J.</b>,
-     <i class='BibTitle'>Semigroups, A package for semigroups and monoids,
-             Version 3.0.20</i>
-     (<span class='BibYear'>2018</span>)<br />
-    (<span class='BibNote'>GAP package</span>),
-    <span class='BibHowpublished'><a href="https://gap-packages.github.io/Semigroups">https://gap-packages.github.io/Semigroups</a></span>.
+    [<span class='BibKey'>Mit18</span>]   <b class='BibAuthor'>J. D. Mitchell et al,</b>,
+     <i class='BibTitle'>Semigroups - GAP package, Version 3.0.20</i>
+     (<span class='BibYear'>2018</span>),<br />
+    <span class='BibHowpublished'><a href="http://dx.doi.org/10.5281/zenodo.592893">http://dx.doi.org/10.5281/zenodo.592893</a></span>.
     </p>
     
 
 bibtex: |
-    @misc{ Semigroups3.0.20,
-      author =           {Mitchell, J.},
-      title =            {{Semigroups}, A package for semigroups and monoids, {V}ersion 3.0.20},
-      month =            {Oct},
-      year =             {2018},
-      note =             {GAP package},
-      howpublished =     {\href                    {https://gap-packages.github.io/Semigroups}
-                          {\texttt{https://gap-packages.github.io/}\discretionary
-                          {}{}{}\texttt{Semigroups}}},
-      keywords =         {transformation semigroups; partial permutations; inverse semigroups;
-                          Green's   relations;   free  inverse  semigroup;  partition  monoid;
-                          bipartitions; Rees matrix semigroups},
-      printedkey =       {Mit18}
+    @manual{ Mitchell2018aa,
```

Comment:
Will this be updated automatically with new version numbers and release dates etc?

Reply:
Nope.  I tried using `{ Mitchell{{site.data.package.date | date: "%Y"}}aa,` as in the citation bit I removed, but the `{{...}}` bit wasn't interpreted.  Not sure if there's a better way, but at the moment this would require manual updating each time.

Reply:
What about the rest of the information in this `@manual` section? Previously it was all updated automatically in the bit you removed, but if needs to be done manually now, then I don't see how that's an improvement.

Reply:
I'm starting to think you're right!

James asked if I could update the website to look like [the PackageManager one](https://gap-packages.github.io/PackageManager/), so I figured I'd do it in the same way – but I'm starting to realise the setup used there is geared towards using ReleaseTools to update everything, and it's not all that applicable to this.  Perhaps a different approach is required...

Reply:
That's very nice of you to do this. Hopefully James will be able to adjust this to get automatic updating into his release process.

Reply:
I'm a very nice man! :D

But I'm starting to think that it'd be better just to change how the thing looks on the surface rather than by changing all these processes (or worse, trying to change the behaviour of `BibEntry`)...

Reply:
Just to clarify: normally with GitHubPagesForGAP, this whole file is autogenerated by the `update.g` script. I have not checked if that is what you are doing, but I hope it is, as that is one of the key ideas behind the whole thing.

Reply:
Our `index.md` contains the following at the bottom to generate the bib entry:
```
<pre style="white-space: pre-wrap;">@manual{ Mitchell{{site.data.package.date | date: "%Y"}}aa,
         Author = { Mitchell, J. D. and others },
         Month  = { {{site.data.package.date | date: "%b"}} },
         Title  = { Semigroups - GAP package, Version {{site.data.package.version}} },
         Doi    = { 10.5281/zenodo.592893 },
         Url    = { http://dx.doi.org/10.5281/zenodo.592893 },
         Year   = { {{site.data.package.date | date: "%Y"}} }
}</pre>
```
Is that not a good idea?

Reply:
Seems fine to me. What I mean above was: "I hope you are indeed running `update.g` to update `_data/package.yml`, instead of, say, manually updating it". Whether you use the bib information in it or not is of course up to you (that bib information is generated by GAP's `Cite` command, and one could certainly think about improving that; all packages using `Cite` to help generated their gh-pages would then immediately benefit from it). But of course in the meantime, if the output of `Cite` does not suite you, I think it's perfectly reasonable to roll your own.

Reply:
I.e., I thought in the discussion above, you guys were discussing whether this `@manual` entry here had to be manually updated; my answer is: "no, it is generated for you". That's all I wanted to say -- sorry if I misunderstood the conversation here :/